weasyprint/css/counters.py

```diff
@@ -295,6 +299,11 @@ def render_marker(self, counter_name, counter_value):
 
         value = self.render_value(counter_value, counter_name=counter_name)
         assert value is not None
+
```

Comment:
You can put the `suffix = …` from line 298 + the return in an `else` block, that would improve readability in my opinion.

Reply:
Will do.